playbooks/provisioning/openstack/post-provision-openstack.yml

```diff
@@ -45,6 +45,14 @@
   become: False
   roles:
     - dns-server-detect
+  tasks:
```

Comment:
This is already handled by the `dns-server-detect` and if it isn't, the `dns-server-detect` role needs to be enhanced as this is the purpose of that role.